build.bat

```diff
@@ -4,7 +4,7 @@ rem Go home.
 cd %~dp0
 
 set PGO_OPT=
-set ARCH_OPT=-x64
```

Comment:
We would like to keep x64 as the default since staying on x32 can cause unexpected issues sometimes, since x32 processes on x64 windows are limited to 2GB of memory.